src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return color.toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constant, use the `name()` method. This is a more reliable way to get the string representation of an enum constant as it directly returns the name of the constant. Refer to the checklist: 'Use name() for getting String representation of enum constants'.